src/index.ts

```diff
@@ -150,3 +152,59 @@ export const major = (version: string | null): number | null => {
 
 	return null;
 };
+
+/**
+ * @summary Check if a version is greater than another
+ * @name lt
+ * @public
+ * @function
+ *
+ * @description Returns true if versionA is less than versionB.
+ * Valid semver versions are always weighted above non semver strings.
+ * Non-semver strings are compared alphabetically.
+ *
+ * @param {string|null} versionA - The version string to compare against
+ * @param {string|null} versionB - The version string to compare to versionA
+ *
+ * @returns {boolean} - true if versionA is less than versionB, otherwise false.
+ *
+ * @example
+ * resinSemver.lt('2.0.5', '1.16.0'); //--> false
+ *
+ * resinSemver.lt('Resin OS 2.0.5', 'Resin OS 2.0.2+rev2'); //--> true
+ *
+ * resinSemver.lt('1.16.0', 'Resin OS 2.0.2+rev2'); //--> true
+ *
+ * resinSemver.lt('Resin OS 2.0.2', 'Resin OS 2.0.2'); //--> false
+ *
+ * resinSemver.lt('Version A', 'Version B'); //--> true
+ */
+export const lt = (versionA: string | null, versionB: string | null): boolean => {
+	if (!versionA && versionB) {
+		return true;
+	}
+
+	if (versionA && !versionB || !versionA && !versionB) {
+		return false;
+	}
+
+	versionA = normalize(<string>versionA);
+	versionB = normalize(<string>versionB);
+
+	const isAValid = semver.valid(versionA);
+	const isBValid = semver.valid(versionB);
+
+	if (!isAValid && !isBValid) {
+		return versionA < versionB;
+	}
+
+	if (!isAValid && isBValid) {
+		return true;
+	}
+
+	if (isAValid && !isBValid) {
+		return false;
+	}
+
+	return semver.lt(versionA, versionB);
```

Comment:
See my [`gte` comment](https://github.com/resin-io-modules/resin-semver/pull/16/files/cc0735a835a2be151755591121674c344f95d415#diff-f41e9d04a45c83f3b6f6e630f10117fe): this should at least be defined in terms of `gte` (e.g. `lt(a, b) => !gte(a, b)`) even if we do want different behaviour to compare, but more likely everything should just be compare-based, I think.